README.md

```diff
@@ -16,54 +12,73 @@ A service for candidates to apply for initial teacher training.
 | QA | [qa.apply-for-teacher-training.education.gov.uk](https://qa.apply-for-teacher-training.education.gov.uk) | For internal use by DfE for testing. Automatically deployed from master |
 
 When setting up a new environment, check you have followed [the instructions
-for doing so](#deploy-env-setup).
+for doing so](/docs/new-environment.md).
 
 ## Table of Contents
 
+* [Documentation](#documentation)
 * [Dependencies](#dependencies)
-* [Prerequisites for development](#dev-prerequisites)
-* [Setting up the development environment](#dev-env-setup)
-* [Setting up new deployment environments](#deploy-env-setup)
-* [Docker workflow](#docker-workflow)
-* [Releases](#releases)
+* [Development environment](#development-environment)
 * [DfE Sign-in](#dfe-sign-in)
-* [Webpacker](#webpacker)
-  * [Debugging Webpacker](#webpacker-debug)
-* [Documentation](#documentation)
-  * [Nomenclature](#documentation-nomenclature)
-  * [Domain Model](#documentation-domain-model)
-  * [Application States](#documentation-application-states)
-  * [Environment Variables](#documentation-env-vars)
-  * [Pipeline Variables](#documentation-pipeline-vars)
-  * [Database Restore](#documentation-db-restore)
-  * [Connecting to live databases](#documentation-db-connect)
-  * [Testing](#testing)
+* [Debugging](#debugging)
+
+## Detailed docs
+
+* [Set up a new environment](/docs/new-environment.md)
+* [Restore a database](/docs/database-restore.md)
+* [Connect to a production database](/docs/connecting-to-databases.md)
+* [Testing style guide](/docs/testing-styleguide.md)
+* [Deploy the application](/docs/deployment.md)
+* [Environment variables](/docs/environment-variables.md)
+* [Pipeline Variables](/docs/pipeline-variables.md)
```

Comment:
Maybe rename this page "Configure availability monitoring" which is all it covers at present?

Reply:
Moved into a separate "Frontend" page (which I have plans for).